### PR TITLE
function to check if link list has cycle

### PR DESCRIPTION
**Pull request by ayush94**
**Adds functionality**  #2921

**Changes:**
a python file added under link_list code that defines a function to check whether a link list has a cycle or not

### CODE REVIEW

File: code/data_structures/linked_list/circular_linked_list/isCircular.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	function checks if a link list has a cycle or not
+	'''
+	try:
+		slow = head
+		fast = head.next
+
+		while slow is not fast:
+			slow = slow.next
+			fast = fast.next.next
+		return true
```
Comment (arnavb) [inline code/data_structures/linked_list/circular_linked_list/isCircular.py]: In Python, true and false are capitalized.

Comment (deesims) [reply code/data_structures/linked_list/circular_linked_list/isCircular.py]: teehee did u test it

Comment (arnavb) [reply code/data_structures/linked_list/circular_linked_list/isCircular.py]: @deesims A quick google search, along with documentation, confirms this. Here is also an [example](https://repl.it/Ngrh/0) in Python 3. Here is one in [Python 2](https://repl.it/Ngrp/0).

Comment (deesims) [reply code/data_structures/linked_list/circular_linked_list/isCircular.py]: yeah i was asking @ayush94 if he tested it, clearly 'true' doesnt parse in python compiler

**Comment by arthurlacoste** [comment]
Don't forgot to [read the contribution guidelines](https://github.com/OpenGenus/cosmos/wiki/contribute) :smile:

Naming convention: folders and filenames should be in **lowercase** and each word **separated by _**. Names should be descriptive to certain extend. 👍

Every code contribution may contain the following comment as a part of it:

````
/* Part of Cosmos by OpenGenus Foundation */
````

**Comment by arnavb** [comment]
@arthurlacoste The comment you mentioned is no longer necessary, and @AdiChat should update the contributing guidelines based on that.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**